core/txpool/blobpool/blobpool.go

```diff
@@ -436,8 +440,6 @@ func (p *BlobPool) Close() error {
 	if err := p.store.Close(); err != nil {
 		errs = append(errs, err)
 	}
-	p.eventScope.Close()
```

Comment:
Tracking the event subscriptions here is not needed anymore since we track them in the txpool now, right?
If so, we should make sure to close the event scope tracker there (p.subs) on `txpool.Close`

Reply:
:+1:, done